app/assets/javascripts/users/promote.js.coffee

```diff
@@ -0,0 +1,6 @@
+$(document).ready ->
+  $ ->
```

Comment:
Right, but you're already inside of a `$(document).ready()` call. You shouldn't then need the `$()` call inside of *that*. That is:

```js
$(document).ready(function(){
  $('#tags').autocomplete({
    //...
  });
});
```

is fine.